scripts/load_timescaledb.sh

```diff
@@ -10,7 +10,10 @@ fi
 # Load parameters - common
 DATA_FILE_NAME=${DATA_FILE_NAME:-timescaledb-data.gz}
 DATABASE_USER=${DATABASE_USER:-postgres}
-
+DATABASE_NAME=${DATABASE_NAME:-astimescaledb}
+DATABASE_HOST=${DATABASE_HOST:-192.168.64.3}
+DATABASE_PORT=${DATABASE_PORT:-30823}
+DATABASE_PWD=${DATABASE_PWD:-aef31a17aae11d04181d5d009}
```

Comment:
Please, do not use your personal values as default here. Instead, let's use the defaults from postgres.

```suggestion
DATABASE_NAME=${DATABASE_NAME:-benchmark}
DATABASE_HOST=${DATABASE_HOST:-localhost}
DATABASE_PORT=${DATABASE_PORT:-5432}
DATABASE_PWD=${DATABASE_PWD:-password}
```

Reply:
Updated